atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresDdlTable.java

```diff
@@ -81,8 +80,7 @@ public void create(byte[] tableMetadata) {
                         ATLASDB_POSTGRES_TABLE_NAME_LIMIT,
                         ATLASDB_POSTGRES_TABLE_NAME_LIMIT);
 
-                final String logMessage = "Failed to create the table {}. " + msg;
-                log.error(logMessage, prefixedTableName, e);
+                log.error("Failed to create the table {}. {}", prefixedTableName, msg, e);
```

Comment:
You should embed the message directly into here if you are doing this otherwise logs will look super weird to people reading them.